test/e2e_node/docker_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = framework.KubeDescribe("Docker features [Feature:Docker]", func() {
+	f := framework.NewDefaultFramework("docker-feature-test")
+
+	Context("when shared PID namespace is enabled", func() {
+		It("processes in different containers of the same pod should be able to see each other", func() {
+			By("Check whether shared PID namespace is enabled.")
+			isEnabled, err := isSharedPIDNamespaceEnabled()
+			framework.ExpectNoError(err)
+			if !isEnabled {
+				framework.Skipf("Skipped because shared PID namespace is not enabled.")
+			}
+
+			By("Create a pod with two containers.")
+			f.PodClient().CreateSync(&v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{Name: "test-pod"},
```

Comment:
nit: let's name it something more distinctive, e.g.,  pid-test-pod, just for our sanity when grepping through logs...